webpack/assets/javascripts/react_app/components/SearchBar/__tests__/__snapshots__/integration.test.js.snap

```diff
@@ -49,7 +9,7 @@ Object {
           "controller": "models",
           "searchQuery": "result",
           "status": "PENDING",
-          "trigger": "ITEM_SELECT",
+          "trigger": "INPUT_CHANGE",
```

Comment:
why the trigger has been changed to `INPUT_CHANGE`?

Reply:
Thanks, actually there was no need for this snapshot even before,
the submit button, triggers the `turbolinks.visit()` which is being tested in this file too,
there is no action or store change after it so no need for a snapshot. the snapshot that was here before was the same as the last snapshot